src/Message/AbstractOrderRequest.php

```diff
@@ -180,7 +180,7 @@ protected function getOrderData()
             'order_tax_amount' => $this->toCurrencyMinorUnits($this->getTaxAmount()),
             'order_lines' => $this->getItemData($this->getItems()),
             'purchase_currency' => $this->getCurrency(),
-            'purchase_country' => $this->getPurchaseCountry(),
+            'purchase_country' => (string) $this->getPurchaseCountry(),
```

Comment:
Why should this be done within the omnipay request? I would prefer type juggling to be done by whatever is creating the data and request

Reply:
I agree with @Feijs here @marleenb90 I think we can close this one. String cast should be done in the client package.